harvard-lts/CURIOSity
# Gitlab aug 18 updates

**Gitlab aug 18 updates**
* * *


# What does this Pull Request do?
The latest gitlab main to GitHub

# How should this be tested?

This is deployed to dev, can go through the changes with someone:

- fix to harvesting to make sure thumbnails come in right
- fix to harvesting so metadata is correct the first time
- fix to harvesting to make sure updates are pulled in
- add urn for FTS
- FTS in UI

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n/a
- integration tests? n/a

# Interested parties
Tag (@ mention) interested parties
